CONTRIBUTORS

```diff
@@ -8,4 +8,6 @@ Google Inc.
  Andrey Konovalov
  David Drysdale
  Baozeng Ding
-Lorenzo Stoakes
+ Lorenzo Stoakes
+SUSE Inc.
+ Jeremy Huang
```

Comment:
Please add "SUSE Inc." to AUTHORS.
AUTHORS are actual copyright holders.
Sorry for the burden, but that's the rules. You need to do it only for the first commit.